doc/src/explanation/special_topics/finite_diff_derivatives.rst

```diff
@@ -59,106 +59,106 @@ In addition to `(x_{i},F_{i})` we next use the two points `(x_{i+1},F_{i+1})` an
 Then we have two equations:
 
 .. math::
-	F_{i+1} = F_{i} + c_{1}* h + \frac{1}{2}*c_{2}*h^2 + \frac{1}{3!}*c_{3}*h^3 + ...
+    F_{i+1} = F_{i} + c_{1}* h + \frac{1}{2}*c_{2}*h^2 + \frac{1}{3!}*c_{3}*h^3 + ...
 .. math::
-	F_{i+2} = F_{i} + c_{1}* (2h) + \frac{1}{2}*c_{2}*(2h)^2 + \frac{1}{3!}*c_{3}*(2h)^3 + ...
+    F_{i+2} = F_{i} + c_{1}* (2h) + \frac{1}{2}*c_{2}*(2h)^2 + \frac{1}{3!}*c_{3}*(2h)^3 + ...
 
 If we again want to find the first derivative (`c_{1}`), we can do that by eliminating the term involving `c_{2}` from
 the two equations.  We show how to do it using SymPy.
 
 
-	>>> from __future__ import print_function
-	>>> from sympy import *
-	>>> x, x0, h = symbols('x, x_0, h')
-	>>> Fi, Fip1, Fip2 = symbols('F_{i}, F_{i+1}, F_{i+2}')
-	>>> n = 3 # there are the coefficients c_0=Fi, c_1=dF/dx, c_2=d**2F/dx**2
-	>>> c = symbols('c:3')
-	>>> def P(x, x0, c, n):
-	...     return sum( ((1/factorial(i))*c[i] * (x-x0)**i for i in range(n)) )
+    >>> from __future__ import print_function
+    >>> from sympy import *
+    >>> x, x0, h = symbols('x, x_0, h')
+    >>> Fi, Fip1, Fip2 = symbols('F_{i}, F_{i+1}, F_{i+2}')
+    >>> n = 3 # there are the coefficients c_0=Fi, c_1=dF/dx, c_2=d**2F/dx**2
+    >>> c = symbols('c:3')
+    >>> def P(x, x0, c, n):
+    ...     return sum( ((1/factorial(i))*c[i] * (x-x0)**i for i in range(n)) )
```

Comment:
This should be no indentation, right?

Reply:
right, but as there was one (of one tab) I keept it (of 4 spaces). TBH I bet I just used `M-x untabify` in emacs...

There's no clear consensus according to:
```bash
$ git grep '^>>> ' doc/**/*.rst | wc -l
227
$ git grep '^ >>> ' doc/**/*.rst | wc -l
19
$ git grep '^  >>> ' doc/**/*.rst | wc -l
1335
$ git grep '^   >>> ' doc/**/*.rst | wc -l
137
$ git grep '^    >>> ' doc/**/*.rst | wc -l
2129
```

From a readability point of view I think I prefer it indented (like literal blocks), which happen to be the most used form according to my greps, so I'm not changing it right now, but tell me if you prefer with no space, I'll change it.

Reply:
If you intent stuff too far in RST it produces a block quote. We actually have block quotes completely disabled in our Sphinx theme because of this, but at some point we should fix everything so that we don't have to.

Reply:
For these, it would mainly be good to have consistency. Probably either no indentation or a single indent. I think base rst requires some indentation with at least an opening `::` but maybe sphinx allows no indentation of code that starts with `>>>`.

Reply:
> it would mainly be good to have consistency

I agree but probably for another PR? It would mean ~3k lines diffing according to my greps (which may be wrong, maybe some of them have a `::`).